MANIFEST.in

```diff
@@ -6,8 +6,8 @@ include aiida/cmdline/templates/*.tpl
 include aiida/common/additions/backup_script/backup_info.json.tmpl
 include examples/testdata/qepseudos/*
 include docs/source/images/*.png
-include fastentrypoints.py
-include setup_requirements.py
+include utils/*
```

Comment:
Don't all python files get automatically included? The `utils` folder only has python files so it should be good no? Or is this necessary because it is outside the `aiida` package folder?